frontend/online-shop/src/theme/MyTheme.tsx

```diff
@@ -0,0 +1,19 @@
+import {createMuiTheme} from '@material-ui/core/styles'
+import '../styles/Fonts.css'
+
+
+const MyTheme = createMuiTheme({
+    palette:{
+        primary:{
+            main: '#a01441'
+        },
+    
```

Comment:
Here also you should remove the empty lines. You will realise that although it will be more condensed it will be easier to read and it will occupy less space. This matters because most applications have big files

Reply:
Fixed.